openreview/conference/templates/bidWebfield.js

```diff
@@ -21,55 +23,111 @@ function main() {
   load().then(renderContent).then(Webfield.ui.done);
 }
 
+function getPapersSortedByAffinity(offset) {
+  if (AFFINITY_SCORE_ID) {
+    return Webfield.get('/edges', {
+      invitation: AFFINITY_SCORE_ID,
+      tail: user.profile.id,
+      sort: 'weight:desc',
+      offset: offset,
+      limit: 50
+    })
+    .then(function(result) {
+      noteCount = result.count;
+      var edgesByHead = _.keyBy(result.edges, function(edge) {
+        return edge.head;
+      });
+      var noteIds = Object.keys(edgesByHead);
+
+      return Webfield.post('/notes/search', {
+        ids: noteIds
+      })
+      .then(function(result) {
+        //Keep affinity score order
+        var notesById = _.keyBy(result.notes, function(note) {
+          return note.id;
+        });
+        notes = noteIds.map(function(id) {
```

Comment:
Is notes declared somewhere else? Or maybe you could just do `return noteIds.map...`

Reply:
oops, my bad. I fixed it.